packages/react-tabs/Spec.md

```diff
@@ -213,18 +229,20 @@ Handling overflow is a built-in feature of tabs and replaceable via a slot.
 
 ### Tab
 
-- TabCommons.value supports an arbitrary identifier for each tab that can be paired with a future tab panel. This is similar to Accordian's selection approach.
+TabCommons.value supports an arbitrary identifier value.
+The value for each tab can be used to display the associated tab panel (future).
+This is similar to Accordian's selection approach.
+
+> TODO: Should value be called key since the tab list has selectedKey?
```

Comment:
I would avoid using `key`, [it has a special meaning that React uses for rendering](https://reactjs.org/docs/lists-and-keys.html). 

IMO `value` is quite clear since there is the selection element in tabs. I guess alternatives could be something like `name` or `tabId` ?

Reply:
I'm OK with updating TabList to have defaultSelectedValue and selectedValue to match with Tab's value property.